mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -209,6 +215,9 @@ final class OneVsRestModel private[ml] (
       newDataset.unpersist()
     }
 
+    var outputColNames = Seq.empty[String]
```

Comment:
Maybe 'predictionColumns' ? These aren't the only output columns. You could make this a mutable val too, but whatever.